### PR TITLE
Update README.md to reflect name of git_config parser

### PR DESCRIPTION
**Pull request by sweiglbosker**
the parser is installed (via ":TSInstall") using the name "git_config" so this might save someone 5 seconds

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [func](https://github.com/amaanq/tree-sitter-func) (maintained by @amaanq)
 - [x] [fusion](https://gitlab.com/jirgn/tree-sitter-fusion.git) (maintained by @jirgn)
 - [x] [Godot (gdscript)](https://github.com/PrestonKnopp/tree-sitter-gdscript) (maintained by @PrestonKnopp)
-- [x] [gitconfig](https://github.com/the-mikedavis/tree-sitter-git-config) (maintained by @amaanq)
+- [x] [git_config](https://github.com/the-mikedavis/tree-sitter-git-config) (maintained by @amaanq)
```
Comment (theHamsta) [inline README.md:225]: The readme list is auto-generated. You must also change parsers.lua. Parsers can specify and alternative name in the readme

Comment (sweiglbosker) [reply README.md]: > The readme list is auto-generated. You must also change parsers.lua. Parsers can specify and alternative name in the readme

Okay, I have updated parsers.lua to reflect the readme.

**Comment by amaanq** [comment]
Parser name is git_config, keep it that way and only adjust the readme name.

**Comment by sweiglbosker** [comment]
> Parser name is git_config, keep it that way and only adjust the readme name.

sorry about my misunderstanding, should be fixed now.

**Review by amaanq — APPROVED** [review]

**Comment by amaanq** [comment]
thanks!

**Pull request merged**